scripts/templates/api.py.mako

```diff
@@ -20,6 +20,7 @@ from templates import helper as th
  """
 import platform
 from ctypes import *
+from ctypes.wintypes import HANDLE
```

Comment:
[NIT] Shouldn't this import be already included in the previous line, and where does `HANDLE` is used? I was thinking maybe if its usage is simple (one line) we could use `wintypes.HANDLE` directly as it should be already imported.